pkg/hostagent/requirements.go

```diff
@@ -137,7 +137,7 @@ A possible workaround is to run "lima-guestagent install-systemd" in the guest.
 
 func (a *HostAgent) optionalRequirements() []requirement {
 	req := make([]requirement, 0)
-	if *a.y.Containerd.System || *a.y.Containerd.User {
+	if *a.y.Containerd.User {
```

Comment:
Isn't `systemd` still required on any OS except Alpine, even for system-containerd?

Reply:
Hmm, might need to check then if there is any other system not using systemd.

But you are right, there should probably be an `isAlpine()` somewhere in there...

Reply:
Hmm, I'm not sure if the Linux distribution is "known" by the time we run these requirements

Maybe it would be OK to let it pass, and then fail the installation if it is not Alpine Linux ?

Reply:
Added a check for alpine